Options.lua

```diff
@@ -39,6 +39,17 @@ local optionsTable = {
                     set = function(info, value)
                         GOW.DB.profile.warnNewEvents = value;
                     end,
+                },
+                hideInCombat = {
```

Comment:
hideInCombat can be capitalized here, other options are like that. or you can lowercase others